src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyRequestDispatcherFilter.java

```diff
@@ -65,12 +69,20 @@ public NextAction handleRead(final FilterChainContext ctx) throws IOException {
         final HttpContent httpContent = ctx.getMessage();
         final HttpRequestPacket request = (HttpRequestPacket) httpContent.getHttpHeader();
 
-        // Handle server disposal
+        // Handle server disposal or initialize (async reconnection)
         if (!requestHandlerProvider.hasHandlerFor(serverAddress)) {
           final HttpResponsePacket.Builder responsePacketBuilder = HttpResponsePacket.builder(request);
           responsePacketBuilder.status(SERVICE_UNAVAILABLE_503.getStatusCode());
           responsePacketBuilder.reasonPhrase(SERVICE_UNAVAILABLE_503.getReasonPhrase());
-          ctx.write(responsePacketBuilder.build());
+
+          String content = "Server not available to handle this request, either not initialized yet or it has been disposed.";
+          responsePacketBuilder.header(CONTENT_TYPE, TEXT.toRfcString());
+          responsePacketBuilder.header(CONTENT_LENGTH, valueOf(content.length()));
```

Comment:
calculate the length of the bytes in the default encoding.
extract it to a constant to avoid recalculating it